package.json

```diff
@@ -9,12 +9,12 @@
     "@types/request": "^2.0.3",
     "@types/run-sequence": "0.0.29",
     "@types/supertest": "^2.0.3",
+    "child_process": "^1.0.2",
```

Comment:
So, you don't need to add this, right? (if you are using built-in child_process module in Node.js)